_topic_maps/_topic_map_osd.yml

```diff
@@ -1426,274 +1426,274 @@ Topics:
   Topics:
   - Name: Serverless overview
     File: about-serverless
----
```

Comment:
We will need to take this title down in Pantheon/Splash Page Manager before this is merged.